CMakeLists.txt

```
@@ -17,6 +17,7 @@ option(USE_SYSTEM_PAJLADA_SETTINGS "Use system pajlada settings library" OFF)
 option(USE_SYSTEM_LIBCOMMUNI "Use system communi library" OFF)
 option(USE_SYSTEM_QTKEYCHAIN "Use system QtKeychain library" OFF)
 option(BUILD_WITH_QTKEYCHAIN "Build Chatterino with support for your system key chain" ON)
+option(USE_SYSTEM_MINIAUDIO "Build Chatterino with support for your system miniaudio" OFF)
```

Nit: more accurately explain what the option does
```suggestion
option(USE_SYSTEM_MINIAUDIO "Build Chatterino with your system miniaudio" OFF)
```